### DISCUSSION TITLE
我感觉我好像要有编制了

### DISCUSSION THREAD

**Discussion by Abandon12393** [share121/inter-knot#3143] [其他]
![359047198-05ee44f2-89c6-4eb8-b1fb-82bef3cf6df9](https://github.com/user-attachments/assets/144f5bce-fe29-4ff1-86ac-80d8a8702540)
在家测了一下以太适应性，爆表了好吧，说不定能去治安局找份工作，挣得肯定不少

**Comment by Abandon12393** [comment]
：）

**Comment by haxyyx** [comment]
治安局，很累的了，建议体验盗洞客，那种激动的，热血的，~~被刺的~~，所以快去体验盗洞客快乐生活吧

**Comment by Lyou777** [comment]
你该不会是在秀吧，未来的王牌调查员就是你了啊！

**Comment by GrunderIndustries** [comment]
# 创立者工业诚邀您加入我司成为测试员!
